pkg/repo/chartrepo_test.go

```diff
@@ -185,3 +189,82 @@ func verifyIndex(t *testing.T, actual *IndexFile) {
 		}
 	}
 }
+
+// StartLocalServerForTests Start the local helm server
+func StartLocalServerForTests(handler http.Handler) (*httptest.Server, error) {
```

Comment:
small nit: would you mind making this private/lowercase if it's not being used outside of the package?